invenio_sipstore/models.py

```diff
@@ -155,6 +175,31 @@ def validate_key(self, filepath, filepath_):
     """Relation to the SIP along which given file was submitted."""
 
 
+class SIPMetadata(db.Model, Timestamp):
```

Comment:
Is this object meant to store different metadata formats per sip, or should we also use it to store metadata edits on the record? If so, how do we keep track of versions